Brandon Benefield

bbenefield89 opened the pull request.

bbenefield89 committed:
WRITE ALBUM SELECT QUERIES

- in setup.sql

bbenefield89 committed:
COMPLETE DAY 1

- in day1.sql

MOVE DAY 1 SQL QUERIES TO `day1.sql`
- in setup.sql => day1.sql

bbenefield89 committed:
CREATE `notes` TABLE

- in notes.sql

bbenefield89 committed:
CREATE `author` TABLE

- in notes.sql

bbenefield89 committed:
CREATE NEW AUTHOR

- in notes.sql

bbenefield89 committed:
CREATE NEW NOTE

- in notes.sql

bbenefield89 committed:
GRAB ALL NOTES BY AUTHOR NAME

- in notes.sql

bbenefield89 committed:
RETURN AUTHOR NAME BY NOTE ID

- in notes.sql

bbenefield89 committed:
FIX QUERIES

- in notes.sql

bbenefield89 committed:
RETURN ALL AUTHORS AND THEIR NUMBER OF NOTES

- in notes.sql

bbenefield89 committed:
CREATE TRANSACTION TO DELETE AUTHOR + AUTHORS NOTES

- in notes.sql

nikhilkamineni commented:
It doesn't look like you wrote any SQL commands that creates tables for the Music Database as described in the instructions for the Day 1 exercises. How are you testing that your queries work?

I'm also getting the following error when trying to run your notes.sql file: ```Error: near line 30: no such column: note.id```

bbenefield89 committed:
QUERIES TO CREATE APPROPRIATE TABLES

- in setup.sql

bbenefield89 committed:
FIX TYPO

- notes.sql
line 31 `where note.id is 1` now refers to `notes.id` instead of `note.id` (plural)

bbenefield89 commented:
Sorry about that everything is fixed :+1:
